README.md

```diff
@@ -4,3 +4,42 @@
 [![Coverage Status](https://coveralls.io/repos/npm/npme-auth-oauth2/badge.svg?branch=master)](https://coveralls.io/r/npm/npme-auth-oauth2?branch=master)
 
 auth strategy for OAuth2 SSO.
+
+# Instructions
+
+> Note: The whitelist file and plugin should both be saved/installed to the Miscellaneous Data Files folder as you've configured it for your npmE instance.
+
+> The default is `/usr/local/lib/npme/data`
+
+## Installation
+
+The change directory command may be different based on your configuration (see note above).
+
+```shell
+cd /usr/local/lib/npme/data
+sudo npm i @bcoe/npme-auth-oauth2-restricted
+
+```
+
+## Configuration
+
+Custom Auth Type and provide the following configuration for `npme-auth-oauth2-restricted`:
```

Comment:
an additional caveat we should add is that the user will need to fill in the OAauth settings for their appliance:

```js
    profileUrl: process.env.OAUTH2_PROFILE,
    emailKey: process.env.OAUTH2_EMAIL_KEY,
    userKey: process.env.OAUTH2_USER_KEY
```

etc., if the user fills these in under the OAuth 2 authentication option, saves, and then switches back to the custom authentication plugin and saves., I believe when I tested I found that the settings are populated appropriately. We can verify this on our testing server.